GitUp/Application/AppDelegate.m

```diff
@@ -411,6 +413,14 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
 #endif
 }
 
+- (void)_getUrl:(NSAppleEventDescriptor*)event withReplyEvent:(NSAppleEventDescriptor*)replyEvent {
+  NSString* urlString = [event paramDescriptorForKeyword:keyDirectObject].stringValue;
+  NSString* githubPrefix = @"github-mac://openRepo/";
+  if ([urlString hasPrefix:githubPrefix]) {
+    [self _cloneRepositoryFromURLString:[urlString stringByReplacingOccurrencesOfString:githubPrefix withString:@"" options:NSCaseInsensitiveSearch range:NSMakeRange(0, urlString.length)]];
```

Comment:
This doesn't make sense: `hasPrefix:` is case sensitive, so doing a case insensitive search and replace afterwards doesn't add anything.